cms/static/sass/_variables.scss

```diff
@@ -240,7 +240,8 @@ $sidebar-color: rgb(246, 246, 246);
 
 // type
 $sans-serif: $f-sans-serif;
-$body-line-height: golden-ratio(.875em, 1);
```

Comment:
NOTE: ``golden-ratio`` is deprecated in the newer version of Bourbon. Moving to ``modular-scale`` (using golden-ratio setting).